### PR TITLE
Fix typo (Windows 8.10 is ever so slightly incorrect)

### PR DESCRIPTION
**Pull request by TheJCAB**
IsWindows8Point10OrGreater to IsWindows8Point1OrGreater. It was subtle.

### CODE REVIEW

**Comment by PRMerger18** [comment]
@TheJCAB : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/versionhelpers/nf-versionhelpers-iswindowsversionorgreater.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 UID: NF:versionhelpers.IsWindowsVersionOrGreater
 title: IsWindowsVersionOrGreater function (versionhelpers.h)
-description: Indicates if the current OS version matches, or is greater than, the provided version information.helpviewer_keywords: ["IsWindowsVersionOrGreater","IsWindowsVersionOrGreater function","base.iswindowsversionorgreater","versionhelpers/IsWindowsVersionOrGreater"]
+description: Indicates if the current OS version matches, or is greater than, the provided version information.
+helpviewer_keywords: ["IsWindowsVersionOrGreater","IsWindowsVersionOrGreater function","base.iswindowsversionorgreater","versionhelpers/IsWindowsVersionOrGreater"]
```
Comment (TheJCAB) [inline sdk-api-src/content/versionhelpers/nf-versionhelpers-iswindowsversionorgreater.md:5]: Note there's an EOL change in there. I did the edit on the github website (using MsEdge), and that was auto-done for me (not sure why).

Comment (Karl-Bridge-Microsoft) [reply sdk-api-src/content/versionhelpers/nf-versionhelpers-iswindowsversionorgreater.md:5]: Yep, the tools do that.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Pull request merged**